sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -641,6 +641,71 @@ describe("LocalWorkspace", () => {
 
         await stack.workspace.removeStack(stackName);
     });
+    it(`runs through the stack lifecycle with an inline program, testing removing without destroying`, async () => {
+        const program = async () => {
+            const config = new Config();
+            return {
+                exp_static: "foo",
+                exp_cfg: config.get("bar"),
+                exp_secret: config.getSecret("buzz"),
+            };
```

Comment:
```suggestion
            class MyResource extends ComponentResource {
                constructor(name: string, opts?: ComponentResourceOptions) {
                    super("my:module:MyResource", name, {}, opts);
                }
            }
            new MyResource("res");
            return {};
```

We need to create a resource in here, otherwise removing the stack will succeed even without the `--force` flag.